indexer/services/l1/service.go

```diff
@@ -260,22 +268,28 @@ func (s *Service) Update(newHeader *types.Header) error {
 			bridgeDepositsCh <- deposits
 		}(bridgeImpl)
 	}
-	go func() {
-		provenWithdrawals, err := s.portal.GetProvenWithdrawalsByBlockRange(s.ctx, startHeight, endHeight)
-		if err != nil {
-			errCh <- err
-			return
-		}
-		provenWithdrawalsCh <- provenWithdrawals
-	}()
-	go func() {
-		finalizedWithdrawals, err := s.portal.GetFinalizedWithdrawalsByBlockRange(s.ctx, startHeight, endHeight)
-		if err != nil {
-			errCh <- err
-			return
-		}
-		finalizedWithdrawalsCh <- finalizedWithdrawals
-	}()
+
+	if s.isBedrock {
+		go func() {
```

Comment:
cc @clabby - we shouldn't be checking the portal at all when Bedrock isn't enabled.